tools/ci_build/build.py

```diff
@@ -525,7 +539,8 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home,
         "-Donnxruntime_USE_NNAPI=" + ("ON" if args.use_dnnlibrary else "OFF"),
         "-Donnxruntime_USE_OPENMP=" + (
             "ON" if args.use_openmp and not args.use_dnnlibrary and
-            not args.use_mklml and not args.use_ngraph and not args.android else "OFF"),
+            not args.use_mklml and not args.use_ngraph and
+            not args.android and not (args.ios and is_macOS()) else "OFF"),
```

Comment:
Lots of `not and`s, shouldn't we just use `or` instead? We could write `x and not y and not z` as `x and not (x or y)`